submissions/TatyanaMolchanova/HTML-CSS-Popup/styles.css

```diff
@@ -0,0 +1,117 @@
+.nav-link img {
+    max-width: 30px;
+}
+.nav {
+    float: right;
+}
+.nav-list {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;  
+    width: 300px;
+    list-style-type: none;
+    padding-right: 30px;
+}
+.nav-item a {
+    text-decoration: none;
+    color: gray;
+}
+
+.popup-link img {
+    max-width: 50px;
+}
+
+#myhtmlpopup {
```

Comment:
Avoid styling by `#id` or HTML tag.
Separate responsibilities:
- HTML tag for markup;
- CSS class for styling;
- Id for JS logic

Reply:
I removed styling by #id.
Thank you for explanation!